conf/messages.en

```diff
@@ -520,6 +520,17 @@ view.write-response-text.continue=Continue
 error.responseText.required=Enter a response to a query from HMRC
 error.responseText.invalid-length=Response must be 1000 characters or fewer
 
+view.amend-case-confirmation.title=You've submitted your extra information
+view.amend-case-confirmation.heading=You've submitted your extra information
+view.amend-case-confirmation.reference=Your case reference number:
+view.amend-case-confirmation.save.heading=Save your case reference number
+view.amend-case-confirmation.save.description=You'll need to use this case reference number for any further correspondence on this case.
+view.amend-case-confirmation.save.description2=It is the same case reference number that you were given when the case was first created.
+view.amend-case-confirmation.next.heading=Next steps
+view.amend-case-confirmation.next.description=Once the information checks are complete, you'll hear from us through CHIEF or your declaration software. We'll let you know if we need any further information from you.
+view.amend-case-confirmation.continue=Send documents for a customs check
+view.amend-case-confirmation.footnote=View guidance on GOV.UK about <a class="govuk-link" href="#">Clearing goods entering, leaving or transiting the UK or EU</a> or <a class="govuk-link" href="#">C1601, C1602, C1603</a> forms.
```

Comment:
Really small thing as well here - "forms" is included within the hyperlink for the C forms. Only noticing this for the first time from looking at this story's screenshot & proto but it's on the other confirmation page so we'll also need the same change for the existing confirmation page (duplicate case page should be covered by the same change)

I think the content spreadsheet is a bit misleading here cuz it has the three links separated and doesn't include form in the link but I double checked again with Meg last week & for now we believe it'll be two links (this may change but for now I've let Irene know and she's gonna update spreadsheet)